### PR TITLE
Add Chronometer by @KevinMarquette

### PR DESCRIPTION
**Pull request by KevinMarquette**

### CODE REVIEW

File: Modules/Chronometer.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Name: Chronometer
+
+## Description:
+Chronometer is a PowerShell script and module profiler that allows you to see line by line execution times of your PowerShell commands.
+
+| Works with | Author(s) | Project URL(s) | Creation Date | Tags |
+|------------|--------|-------------------|---------------|------|
+| PS v5.0+ | Kevin Marquette (<a href="https://kevinmarquette.github.io/blog/?utm_source=PowerShellModulesCentral&utm_medium=markdown" target="_blank">b</a> \| <a href="https://twitter.com/kevinmarquette" target="_blank">t</a>) | [Github](https://github.com/KevinMarquette/Chronometer) | Feb 2017 | Performance, Code Coverage |
```
Comment (ClaudioESSilva) [inline Modules/Chronometer.md]: If you don't mind, please write the complete month name and not only the abbreviation.

**Review by ClaudioESSilva — CHANGES_REQUESTED** [review]
small change

**Comment by ClaudioESSilva** [comment]
Thanks @KevinMarquette!

Just a small change request.
Will do on the other one you submit and to the other modules that are using abbreviations. to maintain consistency

**Comment by KevinMarquette** [comment]
I made that correction

**Review by ClaudioESSilva — APPROVED** [review]
Good to go!

**Comment by ClaudioESSilva** [comment]
Merged.
Thanks again @KevinMarquette !

**Pull request merged**